shiny/_template_utils.py

```diff
@@ -322,3 +322,22 @@ def rename_unlink(file_to_rename: str, file_to_delete: str, dir: Path = app_dir)
         (app_dir / "app-core.py").rename(app_dir / "app.py")
 
     return app_dir
+
+
+def add_test_file(app_dir: Path):
+    test_file = app_dir / "test_app.py"
+    test_file.write_text(
```

Comment:
Let's add a check to be sure the destination file does not exist (and the dest dir _does_ exist)